### PR TITLE
Review request for the times clone.

### PR DESCRIPTION
**Pull request by uchennaanya**
This project is about building an article page cloned from the newyork times article page. 
The features of this project are listed as below:
1. Have an article section
   - With embedded images and videos
   - With embedded section with grid layout
2. Have a Bottom section with
   - More articles section
   - Editor's picks section
   - Most Popular section with aside
3. Newyork times pages nav section
   - Nav links to many other pages aligned in a flex row direction
4. Has a footer with
   - Nav links to Legal, Privacy and Security
   - Copy right information statement

### CODE REVIEW

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
### Request changes 😳
I am very happy to see this project and I can see you worked hard on this project but I am sorry to inform you that I cannot approve this project because it is not completed yet.😟 You need to work more on it. 
_It's okay. We are here to learn. Don't get discouraged. Work even harder. You are almost there.😄_
**Please have a look at the suggestions to get more understanding.**

- [ ] Remove conflict of files

- [ ] Please create only one PR that you use for code review request and submit that one.

- [ ] Please remove all linter errors

- [ ] Add node_modules in gitigonre file.

- [ ] Your Live link is not updated. Please update it.

- [ ] Your PR is not descriptive. Please add more details

- [ ] The whole page is overlapped. Please fix this. 
chrome-capture (22)

- [ ] You have a number of validation errors. Please resolve those too.
 

**Please make the changes and request re-review for the given project**
Stay safe and give me a smile 😇

### Happy Coding
------------------------------------------------------------------------------------------------
**Regards,
Arslan Bisharat
slack: arslanbisharat**

File: assets/css/style.css [hunk 1]
```diff
@@ -0,0 +1,532 @@
...
+  padding: 0.6rem 1rem;
+  border-radius: 4px;
+  font-size: x-small;
+  font-weight: bolder;
+}
+
+.search-btn {
+  border: none !important;
+  color: #000 !important;
+  background-color: transparent !important;
```
Comment (umairarshadbutt) [inline assets/css/style.css]: Remove` !important`

**Comment by umairarshadbutt** [comment]
##
## Pull Request under review by Umair Arshad (TSE)

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<<<<<<< HEAD
+node_modules 
+package-lock.json
+
+=======
+node_modules
+>>>>>>> a98662c6d022f392b0a8d7e1422c35e673489652
```
Comment (umairarshadbutt) [inline .gitignore]: Please remove conflict messages

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi 
Good job with the changes! :tada: We are almost there.
We, however, do have a few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly compare the screenshots below 👇 and work on the suggested changes.
- [ ] Remove `!important `from css file. ([Article for support](https://speckyboy.com/good-bad-css-practices/)) ( [screenshot](https://user-images.githubusercontent.com/29689273/93404399-8997b500-f8a3-11ea-8f88-fb12b8f6ca80.png) from Pathwright)

- [ ] Remove conflict message from .gitignore file 
##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you stick at any point or have any questions, let me know on slack at any time. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Review by arslanbisharat — APPROVED** [review]
### Congratulations APPROVED :clap:
Hello, I can see you worked really hard on this project. and I'm very happy to approve this project with the latest changes you made in this project. :satisfied:
_The clone shows a good understanding and usage of HTML CSS💯_

- The cloned page is rendering as it should and meets all the project specifications.
-  CSS is clean.
-  The markup as well has no errors nor warnings to show. Good job with that.
-  The repo has followed all the guidelines there are to follow.
-  All the required tasks have been fulfilled in this project.
-  Congratulations on the splendid job👍
Now you can merge the branches and move forward with your curriculum. 🎆 

[OPTIONAL] If you wish to delete your currently working branch please update the Live demo link too so that it works properly. 

_Keep working hard._ 

I am very excited to see your next project.

**Happy Coding**:smiley:
------------------------------------------------------------------------------
**Regards,
Arslan Bisharat**
[linkedin](https://www.linkedin.com/in/arslanbisharat/) slack: @arslanbisharat

**Pull request merged**